### DISCUSSION TITLE
Specifying 1.5u row

### DISCUSSION THREAD

**Discussion by darthcircuit** [joshreve/dactyl-keyboard#25] [Q&A]
I'm designing a 6x7 with your repo. I'm trying to make my columns 1.5u on the outer edges of my keyboard. In the run_config.json i see:

```
"pinky_1_5U": true,
    "first_1_5U_row": 0,
    "last_1_5U_row": 5,
```

which i would suspect is what i'm after, but it doesnt seem to do anything. Is there any further documentation on how this works?

![image](https://user-images.githubusercontent.com/1783716/128648610-f8817af3-83df-40bc-99ca-11b029373635.png)

**Comment by itised** [comment]
I had the same question, but I found a comment in generate_configuration.py: https://github.com/joshreve/dactyl-keyboard/blob/b8cb2d942a6287a95e720278bbfe7563352ea48d/src/generate_configuration.py#L59-L65

So it looks like for now it's not an option.

The generate_configuration.py file does have a lot of helpful comments explaining the options - I'd definitely read through it

**Reply by darthcircuit** [reply]
Thank you!

**Reply by joshreve** [reply]
That is exactly what you are looking for and it is in progress.  I got sidetracked while starting to include it.  I'm planning to finish it right after the trackball work.

**Reply by darthcircuit** [reply]
That would be great. Thank you! I'm also looking forward to the track ball so both ￼features are good news for me￼.
